orbit/cmd/orbit/orbit.go

```diff
@@ -219,7 +219,7 @@ func main() {
 		if enrollSecret != "" {
 			const enrollSecretEnvName = "ENROLL_SECRET"
 			options = append(options,
-				osquery.WithEnv([]string{enrollSecretEnvName, enrollSecret}),
+				osquery.WithEnv([]string{enrollSecretEnvName + "=" + enrollSecret}),
```

Comment:
Users are not hitting this error because `orbit` uses `c.String("enroll-secret-path")`, right?

Reply:
Discussed earlier, but this is effecting any new Orbit agent that is trying to enroll because `enroll-secret-path` ends up using the same code path.